worker/pkg/workflows/datasync/activities/gen-benthos-configs/benthos-builder.go

```diff
@@ -1607,14 +1630,45 @@ func shouldProcessFkColumn(t *mgmtv1alpha1.JobMappingTransformer) bool {
 		t.Source != mgmtv1alpha1.TransformerSource_TRANSFORMER_SOURCE_GENERATE_DEFAULT
 }
 
-func constructJsFunction(jsCode, col string) string {
-	if jsCode != "" {
+// func constructTransformJsFunction(jsCode, col string) string {
+// 	if jsCode != "" {
+// 		return fmt.Sprintf(`
+// function fn_%s(value, input){
+//   %s
+// };
+// `, col, jsCode)
+// 	} else {
+// 		return ""
+// 	}
+// }
+
+// func constructGenerateJsFunction(jsCode, col string) string {
+// 	if jsCode != "" {
+// 		return fmt.Sprintf(`
+// function fn_%s(){
+//   %s
+// };
+// `, col, jsCode)
+// 	} else {
+// 		return ""
+// 	}
+// }
```

Comment:
if this isnt needed let's get this comment block out of here.

Reply:
oops good catch, weird that the linter didn't catch that, but removed